### ISSUE TITLE
Is there a way to drop trash?

### ISSUE THREAD

**Issue by jedback** [michaelzangl/minebot#251]
It comes with a lot of cobblestone in inventory when comes back from mining. Is there a way to throw away or in a lava place, for example, all this cobble before store all the other items?

**Comment by biolibi** [comment]
I might be wrong but I don’t think anything like that have been implemented you could probably fill your whole inventory with 1 item in each slot so you don’t pick up trash?

**Comment by jedback** [comment]
No, because I am talking about mine and bring back items to store. Every time that store items, inventory gets clear.

**Comment by michaelzangl** [comment]
The trick is to use /minebot get to put one item of in each of your inventory slots (e.g. 2 slots diamond, 5 hotbar slots pickaxes, 10 coal, iron, ...). That way you don't have to come back from your mining session so early.
